html + sass

Sergiy-Shevchenko opened the pull request.

Sergiy-Shevchenko committed:
start

Sergiy-Shevchenko committed:
html sass

RomarioNik commented on src/partials/section-reviews.html:
```diff
@@ -1,5 +1,49 @@
+        <p class="swiper__text">
+          Every day we come to you with friends after classes in college. We
+          love your ice cream and casual atmosphere!
+        </p>
```
Когда ты прописал "swiper__text" margin-top: 32px, ты текстом отодвигаешь картинку сквозь "swiper__content". Это не совсем правильно, отодвигать марджинами лучше соседей.

Sergiy-Shevchenko replied:
Картинки исправил, марджин повесил на контейнер

RomarioNik commented on src/partials/section-reviews.html:
```diff
@@ -1,5 +1,49 @@
+          <h4 class="swiper__title">Diana, Chicago</h4>
+        </div>
+      </div>
+    </div>
```
Может я ошибаюсь, но вроде слайды должны быть в отдельных дивах. У тебя два последних в одном.

Sergiy-Shevchenko replied:
Сделал слайды - каждый отдельно

RomarioNik commented on src/sass/components/_section-reviews.scss:
```diff
@@ -1,3 +1,43 @@
+    @media screen and (min-width: 320px) {
+        padding: 35px 39px;
+                
+    }
```
25 строка
Здесь падинги можно не ставить в медиа правило, а оставить в общем стиле. Они буду работать до 768px

Sergiy-Shevchenko replied:
правило закомитил

RomarioNik commented on src/sass/components/_section-reviews.scss:
```diff
@@ -1,3 +1,43 @@
+        @media screen and (min-width: 1200px) {
+            padding: 43px 109px;
+        }
+}
```
Текст и заголовок имеют разные размеры на разных устройствах

Sergiy-Shevchenko replied:
не понял комент...

RomarioNik reviewed (changes requested):
По поводу названий картинок, лучше называть так photo@1x.jpg
bryan@1x.jpg
Оно и нагляднее и скорее всего это общепринятая практика)

Sergiy-Shevchenko committed:
reviews

Вже не знаю, що робити з слайдами

1 розмыр адаптуэться з сторонных стилыв

2 крапочки треба якось пыдтягнути вгору

Sergiy-Shevchenko committed:
reviews

Sergiy-Shevchenko committed:
reviews

Sergiy-Shevchenko committed:
reviews

Допилил исправления

Sergiy-Shevchenko committed:
reviews

Фон убрал
Паддинги добавил
Размеры отзовика сделал

Commit:
some changes

RomarioNik committed:
Merge branch 'main' into customer-reviews
